Bottom: cf10daf62bee62378649a1aee71b3cf061c4ffd8
Top:    481cc238687829ef02dcb5d6a9c8d503aadfb318
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-12-19 01:37:48 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
